src/jit/compiler.cpp

```diff
@@ -7202,7 +7225,8 @@ void JitTimer::PrintCsvHeader()
 
     CritSecHolder csvLock(s_csvLock);
 
-    if (_waccess(jitTimeLogCsv, 0) == -1)
+    FILE* fp = _wfopen(jitTimeLogCsv, W("r"));
+    if (fp == nullptr)
```

Comment:
I'm confused: the code already looks like what you've submitted as a PR. You shouldn't submit the same change again. Are you not up-to-date in your source tree?